src/components/measureGroups/SupplementalElements.tsx

```diff
@@ -1,7 +1,381 @@
-import React from "react";
-import { useDocumentTitle } from "@madie/madie-util";
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  MadieDiscardDialog,
+  Toast,
+  AutoComplete,
+} from "@madie/madie-design-system/dist/react";
+import useMeasureServiceApi from "../../api/useMeasureServiceApi";
+import {
+  measureStore,
+  routeHandlerStore,
+  checkUserCanEdit,
+} from "@madie/madie-util";
+import { useFormik } from "formik";
+import { Typography } from "@mui/material";
+import "../editMeasure/measureDetails/MeasureDetails.scss";
+import tw from "twin.macro";
+import { CqlAntlr } from "@madie/cql-antlr-parser/dist/src";
+import { SupplementalData } from "@madie/madie-models";
+import cloneDeep from "lodash/cloneDeep";
+import MeasureGroupAlerts from "./MeasureGroupAlerts";
 
-export default function SupplementalElements(props) {
-  useDocumentTitle(`MADiE Edit ${props.title}`);
-  return <div data-testId={props.dataTestId}>{props.title}</div>;
+const asterisk = { color: "#D92F2F", marginRight: 3 };
+const FormFieldInner = tw.div`lg:col-span-3`;
+const FieldLabel = tw.label`block capitalize text-sm font-medium text-slate-90`;
+const FieldSeparator = tw.div`mt-1`;
+const TextArea = tw.textarea`disabled:bg-slate shadow-sm focus:ring-primary-500 focus:border-primary-500 block w-full sm:text-sm border-gray-500! rounded-md!`;
+
+interface SupplementalElementsProps {
+  title: string;
+  dataTestId: string;
+  setErrorMessage?: Function;
+}
```

Comment:
@RohitKandimalla Originally I was going to keep it to use the same form, but since we are doing separate one for risk adjustment, I'll remove these. Thanks a lot Rohit!